src/tools/rbd_nbd/rbd-nbd.cc

```diff
@@ -779,11 +780,14 @@ class NBDWatchCtx : public librbd::UpdateWatchCtx
              << dendl;
       }
       if (use_netlink) {
-        ret = netlink_resize(nbd_index, new_size);
+        ret = netlink_resize(nbd_index, new_size, cookie);
+        if (ret < 0) {
+          derr << "netlink resize failed: " << cpp_strerror(errno) << dendl;
```

Comment:
`cpp_strerror(errno)` seems bogus here because nothing in `netlink_resize()` sets `errno`.

I think this `if` statement can be dropped entirely as each error in  `netlink_resize()` is logged on its own and with my suggested edits to `derr` log messages there it should be clear where it came from.